tests/test_gist.py

```diff
@@ -223,10 +237,11 @@ def b64encode(s):
 
 
 class TestGistGPG(unittest.TestCase):
+    gnupghome = os.path.abspath('./tests/gnupg')
```

Comment:
This is a style preference, not a necessary change.

Reply:
I moved it for the `tearDownClass` classmethod.